python/sbp/acquisition.py

```diff
@@ -20,7 +20,7 @@
 
 from sbp.msg import SBP, SENDER_ID
 from sbp.utils import fmt_repr, exclude_fields, walk_json_dict, containerize
-from sbp.gnss import GnssSignal, GnssSignalDep
+from sbp.gnss import CarrierPhase, GnssSignal, GnssSignalDep, GPSTime, GPSTimeDep, GPSTimeSec, SvId
```

Comment:
Someone maybe manually edited these files to get them to pass CI?  It looks like they are failing a pyflakes check: 

```
________________________________ pyflakes-check ________________________________
/home/runner/work/libsbp/libsbp/python/sbp/acquisition.py:23: UnusedImport
'sbp.gnss.CarrierPhase' imported but unused
/home/runner/work/libsbp/libsbp/python/sbp/acquisition.py:23: UnusedImport
'sbp.gnss.GPSTime' imported but unused
/home/runner/work/libsbp/libsbp/python/sbp/acquisition.py:23: UnusedImport
'sbp.gnss.GPSTimeDep' imported but unused
/home/runner/work/libsbp/libsbp/python/sbp/acquisition.py:23: UnusedImport
'sbp.gnss.GPSTimeSec' imported but unused
/home/runner/work/libsbp/libsbp/python/sbp/acquisition.py:23: UnusedImport
'sbp.gnss.SvId' imported but unused
```

So, probably need to modify the Python template to ignore that check, or disable pyflakes.  E.g. https://stackoverflow.com/a/10506715/749342 -- so add `  # NOQA` to the end of the import line

Reply:
hmm NVM i think my tox is just broken actually...
`gen python` runs `--remove-all-unused-imports`, had some issues running tox it gets upgraded to tox>4 and returns 'import missing hookimpl', 
instead this was generated running `tox==3.12.1`

works fine running `autoflake -i --remove-all-unused-imports -r python/sbp`